include/kernels/gauge_force.cuh

```diff
@@ -16,7 +16,7 @@ namespace quda {
     const double *path_coeff;
     int count;
 
-    paths(void *buffer, size_t bytes, int ***input_path, int *length_h, double *path_coeff_h, int num_paths, int max_length) :
+    paths(void *buffer, size_t bytes, int pad, int ***input_path, int *length_h, double *path_coeff_h, int num_paths, int max_length) :
```

Comment:
Since I had to get it for `bytes`, then I preferred to not repeat calculations.
On the other hand there is also no reason to have `buffer` and `bytes` outside.
I would suggest then to move everything in `paths` and make the other functions lighter.